qemu/tests/nested_test.py

```diff
@@ -0,0 +1,100 @@
+import os
+import logging
+import tempfile
+
+from avocado.core import data_dir
+from avocado.utils import process
+from avocado.utils import software_manager
+
+from virttest import error_context
+from virttest import data_dir as virttest_data_dir
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Nested test:
+    1) Boot VM
+    2) Install ansible and related packages
+    2) Generate inventory file with L1 guest IP
+    3) Execute ansible command
+    4) Copy test results from L1 to host results directory
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    setup_bridge_sh = "/root/setup_bridge.sh"
```

Comment:
this file in repos, so let's remove copy script function in test. and  add path to playbook.

eg,  network_setup_file:  /var/lib/avocado/data/avocado-vt/test-providers.d/downloads/io-github-autotest-qemu/qemu/deps/setup_bridge.sh

Reply:
Still copy this script now.